src/PactNet/Verifier/IVerifierProvider.cs

```diff
@@ -131,6 +131,14 @@ void AddBrokerSource(Uri url,
                              ICollection<string> consumerVersionSelectors,
                              ICollection<string> consumerVersionTags);
 
+        /// <summary>
+        /// Configures the verifier to return an error
+        /// if no pacts were found when looking up from a broker
+        /// </summary>
+        /// <param name="isError"></param>
```

Comment:
docs: Param docs missing

Reply:
Thanks for review. I've moved it to IPactBrokerOptions and renamed to NoPactsIsError. Also, I've updated xml docs.